src/Extension/MemberExtension.php

```diff
@@ -71,9 +73,36 @@ public function updateCMSFields(FieldList $fields)
             $methodListField->setReadonly(true);
         }
 
+        // Don't show a read only field if the user cannot edit this record
+        // Don't show if a user views their own profile (just let them reset their own password)
+        if ($this->owner->canEdit() && !$this->currentUserIsThisMember()) {
+            $requireNewPassword = CheckboxField::create(
+                'RequirePasswordChangeOnNextLogin',
+                _t(static::class . 'RequirePasswordChangeOnNextLogin', 'Require password change on next login')
```

Comment:
```suggestion
                _t(self::class . '.RequirePasswordChangeOnNextLogin', 'Require password change on next login')
```

LSB in text collection doesn't work =)

Reply:
whoops, got them confused 😄